offices/_vars-offices.html

```diff
@@ -1,8 +1,11 @@
-
+{% set path = '/offices/' %}
 {% set query = queries.initiative %}
-{% set initiatives = query.search_with_url_arguments(size=100,filter_related_office=office.slug) %}
+{% set initiatives = query.search_with_url_arguments(size=6, filter_has_parent="false", filter_related_office=office.slug) %}
+
+{% set tag = "foia" %}
+{% set activity_type = ["cfpb_report"] %}
 
-{% set nav_items = [(path, 'index', office.title)] -%}
+{% set nav_items = [(path + office.slug + '/', 'index', office.title)] -%}
 {% for initiative in initiatives %}
-    {%- set _ignore = nav_items.append((path + initiative.slug+'/', initiative.slug, initiative.title|safe)) -%}
-{% endfor %}
+    {%- set _ignore = nav_items.append(('/initiatives/' + initiative.slug + '/', initiative.slug, initiative.title|safe)) -%}
```

Comment:
Appears to be extra set of parentheses in `append(('/initiatives/' + initiative.slug + '/', initiative.slug, initiative.title|safe))`